src/components/graph/Graph.jsx

```diff
@@ -58,6 +58,10 @@ import utils from "../../utils";
  *      window.alert('Clicked node ${nodeId}');
  * };
```

Comment:
Please also discard changes for `src/components/graph/Graph.jsx`

Reply:
I don't understand this one, you mean I should delete the onDoubleClickNode function ?

Reply:
Please ignore this one 😄